emission/analysis/plotting/geojson/test_geojson.py

```diff
@@ -0,0 +1,27 @@
+import emission.core.get_database as edb
```

Comment:
what does this script do? If it is a utility function, maybe it should be in `bin` instead

Reply:
I added this when I was debugging the data generator. it's just a copy paste from the Timeseries_Sample notebook.

Reply:
ok, so you can either remove it from the commit, or move it into `bin` which is the location for standalone scripts as opposed to modules that are to be integrated into other scripts.